modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -303,7 +303,7 @@ It may take up to 20 minutes for the new member to start.
 In a terminal that has access to the cluster, run the following command:
 +
 ----
-$ oc delete pod -n openshift-config etcd-signer
+$ sudo oc delete pod -n openshift-config etcd-signer
```

Comment:
This `oc delete` should be run from the same master where we created it. The prompt is missing.

```[core@ip-10-0-143-125 ~]$ sudo oc delete pod -n openshift-config etcd-signer```

Reply:
Or simply remove `sudo`. If it is run from a terminal outside of the cluster, then, sudo  should not be added.

Reply:
As discussed on slack, we just removed the sudo command here since this can be deleted from anywhere.